docs/geedocs/answer/7160000.html

```diff
@@ -148,10 +149,95 @@
           <td>Close the resource form and open it again to make the save option available again.</td>
         </tr>
         <tr>
+          <td>340</td>
+          <td>GE Fusion Terrain is black</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
           <td>342</td>
           <td>Fusion crashes when opening an unsupported file type</td>
           <td>Re-open fusion and avoid opening unsupported file types.</td>
         </tr>
+        <tr>
+          <td>375</td>
+          <td>unbundle psycopg2</td>
+          <td>On Ubuntu 16.04 switch to python-psycopg2' instead of 'python2.7-psycopg2'.</td>
+        </tr>
+        <tr>
+          <td>380</td>
+          <td>Provider field in resource-view is blank</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
+          <td>401</td>
+          <td>GEE commands are not in the path for sudo. </td>
+          <td>Add /opt/google/bin to the path for all users, including the super user</td>
+        </tr>
+        <tr>
+          <td>402</td>
+          <td>Provider manager window locked to main window.</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
+          <td>403</td>
+          <td>Missing close button on system manager window in RHEL 7</td>
+          <td>Right click the title bar and select close</td>
+        </tr>
+        <tr>
+          <td>404</td>
+          <td>Opaque polygons in preview.</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
+          <td>405</td>
+          <td>Vector layer preview not cleared in some situations</td>
+          <td>Reset the preview window to the correct state by either clicking on it or previewing another vector layer</td>
+        </tr>
+        <tr>
+          <td>407</td>
+          <td>Corrupt data warning when starting fusion</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
+          <td>423</td>
+          <td>Improve JPEG 2000 format support performance</td>
+          <td>Use Geotiff or other image formats.</td>
+        </tr>
+        <tr>
+          <td>437</td>
+          <td>Rebooting VM while it is building resources results in a corrupted XML</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
+          <td>440</td>
+          <td>Fuzzy imagery in historical imagery tests.</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
+          <td>444</td>
+          <td>Fusion installer does not upgrade the asset root on RHEL 7</td>
+          <td>Upgrade the asset root manually by running the command that is printed when you try to start the fusion service</td>
+        </tr>
+        <tr>
+          <td>453</td>
+          <td>Improve `check_server_processes_running` detection for uninstall</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
+          <td>474</td>
```

Comment:
We should also add issue #456 in the known bugs.

Reply:
Ok. I'll add it as well to the ticket.